web/src/components/auth/UserMenu.vue

```diff
@@ -214,6 +214,7 @@ export default class UserMenu extends Vue {
   async logout() {
     await this.$store.dispatch(AuthActions.Logout);
     this.open = false;
+    this.$router.push('/');
```

Comment:
Not sure if we always want to do this. It’s probably fine for now but expect this to be reported as a bug eventually

Reply:
Yeah, what was happening was that the `moderator` logged out but still they were on the moderator dashboard so i decided to redirect on logout however that can be changed